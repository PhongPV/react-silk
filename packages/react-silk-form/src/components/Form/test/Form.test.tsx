import * as React from 'react'
import TestRenderer from 'react-test-renderer'

import { Form } from '../index'

describe('<Form />', () => {
  test('renders without errors', () => {
    const text = TestRenderer.create(<Form />)
    expect(text.toJSON()).toBeTruthy()
  })

  test('match snapshot', () => {
    const text = TestRenderer.create(<Form />)
    expect(text.toJSON()).toMatchSnapshot()
  })
})
