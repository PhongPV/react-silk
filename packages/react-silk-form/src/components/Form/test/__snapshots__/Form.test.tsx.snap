// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> match snapshot 1`] = `
<div>
  <label>
    Form
  </label>
  <input
    type="text"
  />
</div>
`;
